src/vicsetup/libvicsetup/crypto.h

```diff
@@ -0,0 +1,57 @@
+#ifndef _VIC_CRYPTO_H
+#define _VIC_CRYPTO_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include "vic.h"
+
```

Comment:
Some comments documenting what the API calls do would be nice.

Reply:
Perhaps but this is an internal API. Vicsetup provides only one public API used by SGX-LKL: **libcryptsetup.h** and this is documented elsewhere.

Reply:
As long as the expectation is that no one will ever modify the vicsetup code, that's fine.  If we expect vicsetup to be maintained, please add doc comments for all functions.